src/components/AccessForm.vue

```diff
@@ -73,24 +78,48 @@ export default defineComponent({
             trigger: "change"
           }
         ]
-      }
+      },
+      loading: false
     };
   },
+  computed: {
+    ...mapState(["ourSecretKey", "decryptedMessage"]),
+    ...mapGetters(["enclavePublicKey"])
+  },
   methods: {
+    ...mapActions(["sealBox"]),
+    ...mapMutations(["setDecryptedMsg"]),
     onSubmit(formName: string) {
       (this.$refs[formName] as typeof elForm).validate(
         async (valid: boolean) => {
           if (valid) {
-            const enclavePublicKey = this.$store.getters.enclavePublicKey;
+            this.loading = true;
```

Comment:
I might be missing it, but is this set to false after the calls?

Reply:
Oh yes, since I call setting it to false on `finally`. 
I forgot to update it after changing to async await.